### DISCUSSION TITLE
[Feature] Metadata view while searching through duplicates

### DISCUSSION THREAD

**Discussion by tqqk** [immich-app/immich#10363] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Include photos' info (metadata), while deciding which one should I delete. 

Lot of my photos in duplicates have wrong date taken, because of an error I made years ago, so it would be great if I could see the image metadata while deciding which picture should I delete.

Feature preview:
![immich-feature-request](https://github.com/immich-app/immich/assets/93881136/b735a89b-3df7-4cbf-9cf1-052744c974cd)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by blublub** [comment]
+1 - at least for the most relevant metadata, size, date, resolution, camera, date of creation and last changed

**Comment by Coderdude112** [comment]
Metadata that differs could be shown at the top, and differences highlighted similar to a git diff

**Comment by MrColumbo** [comment]
i have a lot of duplicates where one is geo tagged and the other is not.  I like to remove the ones which do no contain geo metadata ....this feature will help me accomplish this task :)

**Comment by Coderdude112** [comment]
There's another project that might have this functionality, but I haven't been able to get it to work.
If anyone is successful, ping me.
https://github.com/vale46n1/immich_duplicate_finder
